opensearch-observability/src/test/kotlin/org/opensearch/observability/PluginRestTestCase.kt

```diff
@@ -165,9 +168,13 @@ abstract class PluginRestTestCase : OpenSearchRestTestCase() {
             exception.response
         }
         if (expectedRestStatus != null) {
+//            println("response is $response")
```

Comment:
Can we remove any commented-out lines of they are not necessary?

Reply:
yeah okay. I'll remove them